wfsim/strax_interface.py

```diff
@@ -307,11 +486,11 @@ def __init__(self, config):
                  help="Directory with fax instructions"),
     strax.Option('fax_config_override', default=None,
                  help="Dictionary with configuration option overrides"),
-    strax.Option('event_rate', default=5, track=False,
+    strax.Option('event_rate', default=1000, track=False,
                  help="Average number of events per second"),
     strax.Option('chunk_size', default=100, track=False,
                  help="Duration of each chunk in seconds"),
-    strax.Option('nchunk', default=1, track=False,
+    strax.Option('nchunk', default=10, track=False,
                  help="Number of chunks to simulate"),
     strax.Option('right_raw_extension', default=50000),
     strax.Option('timeout', default=1800,
```

Comment:
I think timeout does not have to be tracked

Reply:
I'm pretty sure it's not even needed anymore. It's from back in the day when the mailbox timed out after 1 minute or so and assumed everything crashed